src/App.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import Layout from './components/Layout';
+import Splash from './pages/Splash';
+import TradeDetails from './pages/TradeDetails';
+import Reservations from './pages/Reservations';
+import ReserveFromTd from './pages/ReserveFromTd';
+import Reserve from './pages/Reserve';
+import AddTrades from './pages/AddTrades';
+import DeleteTrade from './pages/DeleteTrade';
+import ListTrades from './pages/ListTrades';
+import './App.css';
+import Login from './components/auth/Login';
+import Register from './components/auth/Register';
+import NotFound from './components/NotFound';
+
+const App = () => (
```

Comment:
- [ ] When I try to run the frontend app according to instructions, this happens:

![image](https://github.com/lincoln1883/appointment-capstone-frontend/assets/57421823/5fcefdcf-252e-4f21-aa89-92a305cf50d9)

![image](https://github.com/lincoln1883/appointment-capstone-frontend/assets/57421823/c809cbbd-4ca3-4bf6-8b51-94424b7de341)

This is because your project depends on environmental variables on a .env file that is not being shared:

![image](https://github.com/lincoln1883/appointment-capstone-frontend/assets/57421823/c5a0b53a-a71e-4d6c-a448-2f3816390b97)

![image](https://github.com/lincoln1883/appointment-capstone-frontend/assets/57421823/78987638-c4a3-4265-9c5b-f43efc27ceb4)

Kindly, add specific instructions on your readme about how to setup .env file with the correct variables/attributes.  

That way, all your users will be able to setup and run your project smoothly. 👍 

_note: I'll create my own .env in order to be able to continue the review._

Reply:
Added instruction for .env file in the readme file.